lib/publishing_api_payload.rb

```diff
@@ -112,4 +116,23 @@ def roles_and_people(role_appointments)
         memo[:people] = (memo[:people] + people).uniq
       end
   end
+
+  def attachments
+    file_attachments = edition.file_attachment_revisions
+
+    file_attachments.map do |attachment|
+      FileAttachment.new(attachment, edition.document).payload
+    end
+  end
+
+  def featured_attachments
+    file_attachments = edition.file_attachment_revisions
+
+    file_attachments.map do |attachment|
+      {
+        content_type: attachment.content_type,
+        content: attachment.asset.bytes,
```

Comment:
I was a bit surprised that we were sending binary or string data here 😱 - I believe the intention is this would have content_type: "text/govspeak" and content be a govspeak snippet. Though I'm hoping to change this to an id and have components directly rendered on the frontend.

Reply:
To be fair, I struggled to find documentation about what should go in this field. I was hoping a review would provide clarity 😄